tests/unit/util/json_util_test.py

```diff
@@ -79,6 +79,9 @@ def test_long(self):
     def test_negative_long(self):
         self.__test_encode_decode(r"-1234567890123456789", -1234567890123456789)
 
+    def test_long_long_int(self):
+        self.__test_encode_decode("18446744073709551615", 18446744073709551615)
```

Comment:
This check should be fine and cover most of the scenarios, but perhaps to be on the safe we could also add explicit test case for orjson - ensure that it throws overflow error on large int and verify our wrapper works correctly (although, yeah, the second part should already be covered by this test case).

Reply:
Added orjson.dumps error catch assertions.